content/src/roadmaps/task-dependencies.json

```diff
@@ -28,186 +28,188 @@
       ]
     },
     {
-      "name": "floor-plan-approval-doh",
-      "dependencies": [
+      "task": "floor-plan-approval-doh",
+      "taskDependencies": [
         "sign-lease"
       ]
     },
     {
-      "name": "individual-staff-licenses-health-aide",
-      "dependencies": [
+      "task": "individual-staff-licenses-health-aide",
+      "licenseTaskDependencies": [
         "home-health-aide-license"
       ]
     },
     {
-      "name": "site-safety-permits-cosmetology",
-      "dependencies": [
+      "task": "site-safety-permits-cosmetology",
+      "taskDependencies": [
         "sign-lease",
         "building-permit"
       ]
     },
     {
-      "name": "town-mercantile-license-liquor",
-      "dependencies": [
+      "task": "town-mercantile-license-liquor",
```

Comment:
`licenseTask`?

Reply:
Unclear? Their is a town-mercantile-license.md in both the license-task and tasks folder. I'll follow up with content to make sure they're using the right one. Good catch. 

![image](https://github.com/newjersey/navigator.business.nj.gov/assets/22485301/24cb0eda-4119-41ef-9340-6db48638a213)
![image](https://github.com/newjersey/navigator.business.nj.gov/assets/22485301/e3d361f2-4e0d-41a0-805f-434c9a552d5e)

Reply:
JK, so it looks t o me like the liquor verision is part of task? 
![image](https://github.com/newjersey/navigator.business.nj.gov/assets/22485301/4af8eb54-a6f2-43be-a5e8-cd112cead630)

Reply:
Ignore my first comment, now deleted, I can't read.

Reply:
Also, there is a `town-mercantile-license-liquor.md` in the **tasks** folder, but I think it should be in **license-tasks** folder & I am wondering why it isn't. 
![Screenshot 2023-07-12 at 1 19 02 PM](https://github.com/newjersey/navigator.business.nj.gov/assets/75342585/ccbf9801-476c-4621-9d4f-687b183e4dac)

Reply:
So I know that Anne told me the answer to this question before

and I think it has to do with webflow syncs and some content not being ready to be moved from a task to a license task as we only support 1 of those 2 things for webflow syncs and their is like extra content that needs to be added before we can move it. 

I'm not super confident in that answer though so perhaps we should rope her into the conversation for a re-fresher @aloverso

Reply:
Also, along the lines of `town-mercantile-license`, we should update the key to `licenseTask`.
![Screenshot 2023-07-12 at 1 29 40 PM](https://github.com/newjersey/navigator.business.nj.gov/assets/75342585/4273158f-3646-49f8-93a5-2378dacb38a5)

Reply:
So I believe there are 2 town-mercantile-license files, one in task and one in license task so I think its ambigious as to which the content team actually wants so I will ask.

Reply:
Vera mentioned its probably Task but she will check again and modify once its live on dev